feg/protos/s8_proxy.proto

```diff
@@ -176,25 +179,29 @@ message DeleteSessionResponsePgw{
 }
 
 message CreateBearerRequestPgw {
-    uint32 c_agw_teid = 1;
-    uint32 linked_bearer_id = 2;
-    ProtocolConfigurationOptions protocol_configuration_options = 3;
-    BearerContext bearer_context = 4;
-    bytes indication_flag = 5;
+    string pgwAddrs = 1;
+    uint32 sequence_number = 2;
+    uint32 c_agw_teid = 3;          //teid given by FEG during CreateSessionm
```

Comment:
Comment should be on the line above `uint32 c_agw_teid = 3;` so that the comment is added into the generated files

also space between `//` and comment